README.md

```diff
@@ -57,37 +57,16 @@ To get the source code from GitHub, type
     cd ./azure-sdk-for-php
 
 > **Note**
-> 
+>
 > The PHP Client Libraries for Microsoft Azure have a dependency on the [HTTP_Request2](http://pear.php.net/package/HTTP_Request2), [Mail_mime](http://pear.php.net/package/Mail_mime), and [Mail_mimeDecode](http://pear.php.net/package/Mail_mimeDecode) PEAR packages. The recommended way to resolve these dependencies is to install them using the [PEAR package manager](http://pear.php.net/manual/en/installation.php).
 
-
 ##Install via Composer
+1. Make sure you have installed **[Compoeser](https://getcomposer.org/doc/00-intro.md#introduction)** in your system.
+It can be installed locally or globally depending on your own needs.
 
-1. Create a file named **composer.json** in the root of your project and add the following code to it:
```

Comment:
There were 2 problems when I ran you command. 
First, I needed to add .phar to composer. The command that works for me was "php composer.phar create-project microsoft/windowsazure ./test ^0.4". Once it worked, it pulled down an older build that did not work because of missing dependencies. 
Second, once I ran "php composer.phar create-project microsoft/windowsazure ./test2 dev-master", it pulled down the dev-master, but WindowsAzure folder is not part of the vendor folder. I don't think that is what we want the end users to see, especially for beginners. 

The readme file is mostly for beginners and we want to make it very simple to follow. 

Thanks a lot for your contribution. 
Yaqi